### ISSUE TITLE
Broken import for pgtype

### ISSUE THREAD

**Issue by amoiseiev** [kyleconroy/sqlc#2417]
### Version

Other

### What happened?

sqlc 1.9.0

sqlc generates the following import in certain scenarios:

"github.com/tabbed/pqtype"

while leads to the following error running `go mod tidy`

```
    github.com/tabbed/pqtype: github.com/tabbed/pqtype@v0.2.0: parsing go.mod:
    module declares its path as: github.com/sqlc-dev/pqtype
            but was required as: github.com/tabbed/pqtype
```

Probably related to the namespace migration.

### What operating system are you using?

macOS

### What database engines are you using?

PostgreSQL

### What type of code are you generating?

Go

**Comment by andrewmbenton** [comment]
Yep, this was overlooked when we moved sqlc-related things into the sqlc-dev GitHub organization. I'll have a PR up soon to fix this, and we may cut a new release. I haven't thought about it too hard, but you may be able to work around this with a [replace directive](https://go.dev/ref/mod#go-mod-file-replace) in your go.mod file.

**Comment by andrewmbenton** [comment]
This should be resolved as of the [v1.19.1 release](https://github.com/kyleconroy/sqlc/releases/tag/v1.19.1) today.
